### ISSUE TITLE
Expression class specific functions require additional explanation material

### ISSUE THREAD

**Issue by CarlGustavAlbertDwarfsteinYung** [godotengine/godot-docs#4332]
**Your Godot version:**
3.2.3

**Issue description:**

Expression class allows parsing of a mathematical expression with the ability to use some basic functions as well.  These functions are not documented anywhere forcing me to look though the source although I was not able to make some of them work. 
https://github.com/godotengine/godot/blob/27763b67bb63139319bf611590c40e48663e72d6/core/math/expression.cpp#L1094

Please provide an additional reference page for Expression class explaining what functions are available and how to use them.  

**URL to the documentation page (if already existing):**
https://docs.godotengine.org/en/stable/classes/class_expression.html?highlight=expression

**Comment by Calinou** [comment]
I suppose the list of valid functions should be a dedicated documentation page that can then be linked to in the Expression class' Tutorials section.

**Comment by CarlGustavAlbertDwarfsteinYung** [comment]
Thank you. I hope that didn't come out too strong like I'm making a demand. I should have said it slightly different I guess. Maybe "it would be nice to have a reference." I have tested some functions and some don't work exactly same as their GScript equivalent or may even have different naming. For example `color_name("red", 1)` vs gdscript ColorN("red") alpha is optional. They require their own explanation. Even how to pass variables which I have figured out how to do does also require a bit of explaining :)
